src/Sentry.Bindings.Android/Sentry.Bindings.Android.csproj

```diff
@@ -1,6 +1,6 @@
 <Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0-android</TargetFramework>
+    <TargetFrameworks>net7.0-android</TargetFrameworks>
```

Comment:
would be a breaking change but had to do this to get things to compile. We can try getting back to 6 before merging

Reply:
Similar to this:

 > Error: /Users/runner/.dotnet/sdk/8.0.100-rc.1.23455.8/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Sdk.ImportWorkloads.targets(38,5): error NETSDK1147: To build this project, the following workloads must be installed: wasm-tools-net6 [/Users/runner/work/sentry-dotnet/sentry-dotnet/src/Sentry.Maui/Sentry.Maui.csproj::TargetFramework=net6.0-ios]